packages/members-api/lib/migrations/index.js

```diff
@@ -98,4 +101,120 @@ module.exports = class StripeMigrations {
             }
         }
     }
+
+    async findPriceByPlan(plan) {
+        const currency = plan.currency ? plan.currency.toLowerCase() : 'usd';
+        const amount = Number.isInteger(plan.amount) ? plan.amount : parseInt(plan.amount);
+        const interval = plan.interval;
+
+        const price = await this._StripePrice.findOne({
+            currency,
+            amount,
+            interval
+        });
+
+        return price;
+    }
+
+    async populateStripePricesFromStripePlansSetting(plans) {
+        const stripeProductsPage = await this._StripeProduct.findPage({limit: 1});
+        const defaultStripeProduct = stripeProductsPage.data[0];
+
+        if (!defaultStripeProduct) {
+            this._logging.info('Could not find Stripe Product');
```

Comment:
I think we need to return from here if the product is not found, or it will fail later in the method

Reply:
Actually, instead of return, we might need to create a default Stripe Product here if missing, as for existing or new sites there won't be a Stripe Product in DB unless there is a subscription which exists.